requirements.txt

```diff
@@ -1,7 +1,9 @@
-jsonschema>=2.5.1
-flask>=0.10.1
-PyYAML>=3.11
-requests>=2.9.1
-six>=1.7
-strict-rfc3339>=0.6
-swagger_spec_validator>=2.0.2
+jsonschema>=2.5.1,<3
+flask>=0.10.1,<0.12
+PyYAML>=3.11,<4
+requests>=2.9.1,<3
+six>=1.7,<2
+strict-rfc3339>=0.6,<0.7
```

Comment:
- It allows `connexion` to benefit from fixes eg `flask` have now released `0.11.1`.  
- Generally flexibility is desirable (where it doesn't break things!) eg maybe my application really needed the bug fix in `flask 0.11.1`: then I'd have been annoyed if `connexion` had insisted on exactly `0.11.0`.